AppTelemetry/src/oteladapter.cxx

```diff
@@ -0,0 +1,43 @@
+/*
+ * (c) 2024 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+ *
+ * RTI grants Licensee a license to use, modify, compile, and create derivative
+ * works of the Software.  Licensee has the right to distribute object form
+ * only for use with RTI products.  The Software is provided "as is", with no
+ * warranty of any type, including any warranty for fitness for any purpose.
+ * RTI is under no obligation to maintain or support the Software.  RTI shall
+ * not be liable for any incidental or consequential damages arising out of the
+ * use or inability to use the software.
+ */
+#include <memory>
+
+#include "oteladapter.hpp"
+#include "otelconnection.hpp"
+#include <rti/routing/Logger.hpp>
+
+using namespace rti::routing;
+using namespace rti::routing::adapter;
```

Comment:
You're using the full namespace below. You can try to normalize it to using rti::routing::adapter everywhere or getting rid of rti::routing::adapter:: in the StreamReaderListeners below.